aspnetcore/web-api/index/samples/6.x/Program.cs

```diff
@@ -0,0 +1,38 @@
+#define GLOBAL // FIRST GLOBAL
+#if NEVER
+#elif FIRST
+#region snippet1
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddControllers();
+
+var app = builder.Build();
+
+app.UseHttpsRedirection();
+
+app.UseAuthorization();
+
+app.MapControllers();
+
+app.Run();
+#endregion
+#elif GLOBAL
+#region snippet_global
+using Microsoft.AspNetCore.Mvc;
+[assembly: ApiController]
```

Comment:
@serpent5  to verify this is working, I created the `WeatherForecast2Controller ` which derives from controller, not `ControllerBase` and doesn't use `[Route("[controller]")]`.  Therefore WeatherForecast2Controller  doesn't require routing and compiles fine with `#elif FIRST`. But compiling the snippet_global I get a run time error that it doesn't contain routing.